bot.py

```diff
@@ -13,6 +13,8 @@
 
 load_dotenv()
 TOKEN = os.getenv('DISCORD_TOKEN')
+if TOKEN is None:
+    raise Exception("No DISCORD_TOKEN found. Be sure you have an .env file with a DISCORD_TOKEN defined")
```

Comment:
They don't *have* to use a .env file - really they just have to have an environment var, and `dotenv` makes that portable and easy for testing without having to constantly muck with your env vars